src/components/03-components/03-buttons/01-button/_button--single.njk

```diff
@@ -0,0 +1,68 @@
+{#
+    Creates a button.
+    link        (string) - use "a" instead of "button". String should be any valid HREF value
+    modifiers   (array)  - USA "secondary,base,outline,inverse,big,unstyled,hover,active,focus"
+    smodifiers  (array)  - SDS "circle,small"
+    disabled    (bool)   - adds disabled attribute 
+    classes     (string) - extra classes
+    ext         (string) - other attributes i.e. ( tabindex="0" onclick="alert" )
+    text        (string) - text of the button 
+    icon        (object) - icon before text if there is text {prefix,icon,title} 
+    icon_right  (object) - icon after text if there is text {prefix,icon,title} 
+#}
+<{%
+if link 
+%}a href="{{link}}{% 
+else %}button{% 
+endif 
+%} class="usa-button{%
+if modifiers 
+%}{%
+    for modifier in modifiers 
+    %}{% if modifier !== 'default' %} usa-button--{{ modifier }}{%  endif %}{%
+    endfor 
+%}{% 
+endif 
+%}{%
+if smodifiers 
+%}{%
+    for modifier in smodifiers 
+    %} sds-button--{{ modifier }}{%
+    endfor 
+%}{% 
+endif 
+%}{%
+if classes
+%} {{classes}}{%
+endif
+%}"{%
+if ext
```

Comment:
Where is this used and what does it do? It's not an attribute for buttons or a tags